compiler/test/dotty/tools/dotc/FromTastyTests.scala

```diff
@@ -24,28 +24,104 @@ class FromTastyTests extends ParallelTesting {
     implicit val testGroup: TestGroup = TestGroup("posTestFromTasty")
     val (step1, step2) = compileTastyInDir("../tests/pos", defaultOptions,
       blacklist = Set(
+        /* assertion failed: owner discrepancy for type CC, expected: class <refinement>, found: class <refinement>
+            at scala.Predef$.assert(Predef.scala:219)
+            at dotty.tools.dotc.core.tasty.TreeUnpickler$TreeReader.symbolAtCurrent(TreeUnpickler.scala:195)
+         */
```

Comment:
What's the motivation for adding error message in the code?

Reply:
Easier to know which is the reason for the failure without rerunning all of them each time. We also had that in the stib blacklist.

Reply:
I think it would make more sense to open issues for each of them.

Reply:
I will work on minimizing them and open an issue with the minimized code. Then would refer to the group of tests that is likely to be failing because of that particular issue.